ex7.8_NumPy_replace_array.py

```diff
@@ -0,0 +1,26 @@
+import numpy
+import numpy as np
+
+
+def replace_number(matrix: numpy.ndarray, number: int, sign: chr):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/